### ISSUE TITLE
Modal: Option to remove OK button from tab order on modal on list

### ISSUE THREAD

**Issue by ssaniel** [infor-design/enterprise#5358]
**Is your feature request related to a problem or use case? Please describe.**
On a list with allow update on a code block column:

1. Tab or click the cell to launch the code block modal
2. The OK button should be taken out of the tab order (so should not be able to tab to it)
https://jira.lawson.com/browse/LMCLIENT-35634

**Describe the solution you'd like**
Option to remove OK button from tab order on modal on list

**Additional context**
![image](https://user-images.githubusercontent.com/78124324/123235080-201d9480-d50e-11eb-8369-599792606ba5.png)

**Comment by tmcconechy** [comment]
It seems like a strange use case to remove the ability to use the keyboard to close the modal. So fundamentally not sure if this is a good idea.

Whats the actual reason to not tab to the button?

If you want this can you just set the tabindex = -1 or disable the button? As this doesnt seem like a feature i would like to promote

**Comment by ssaniel** [comment]
ah yes, that makes sense. will try that out and get back to you. Thanks Tim!

**Comment by vonnyw** [comment]
@tmcconechy Requesting this to be prioritized and patched for IDS 4.52 as it's hurting the power user experience when entering code block values. We are asking for a  modal setting to set the tabIndex for the OK button to be -1.

**Comment by tmcconechy** [comment]
OK, found this was rather easy so not going to complain that this seems wrong for accessibility to remove the tabindex. I'll add it but why does this really need a patch or can you use the modal events to change the tabindex on the modal. Seems like you should be able to and it seems like your violating accessibility principals.

**Comment by tmcconechy** [comment]
Also you could setup the modals like this https://github.com/infor-design/enterprise/blob/main/app/views/components/modal/example-full-content.html where you spec the full markup. Then it would be possible to just set the tabindex.

**Comment by vonnyw** [comment]
Thanks Tim, @ssaniel  please try Tim's suggestions and report back if it works. This short term change addresses the lack of efficiency in code block entry for power user, there will be 2 modes eventually after the accessibility setting is rolled out and communicated.

**Comment by tmcconechy** [comment]
ok the setting was simple. So its in If needed but i agree lets try that.

**Comment by jbrcna** [comment]
this issue is now resolved. moving this ticket to Done
v4.53.0-dev
verified in https://main-enterprise.demo.design.infor.com/components/modal/test-no-tabindex.html
